source/extensions/extensions_metadata.yaml

```diff
@@ -95,6 +95,11 @@ envoy.clusters.redis:
   - envoy.clusters
   security_posture: requires_trusted_downstream_and_upstream
   status: stable
+envoy.clusters.logical_dns:
+  categories:
+  - envoy.clusters
+  security_posture: requires_trusted_downstream_and_upstream
```

Comment:
Is this the right posture? I was under the impression that as part of core clusters it was "robust_to_untrusted_downstream_and_upstream", but maybe I'm wrong.

Reply:
good catch thanks, fixed.